src/components/executive-education-2u/EnrollmentCompleted.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { Redirect } from 'react-router-dom';
+import { Container } from '@edx/paragon';
+import { Helmet } from 'react-helmet';
+import { AppContext } from '@edx/frontend-platform/react';
+import PropTypes from 'prop-types';
+import CourseSummaryCard from './components/CourseSummaryCard';
+import EnrollmentCompletedSummaryCard from './components/EnrollmentCompletedSummaryCard';
+
+const EnrollmentCompleted = (props) => {
+  const { enterpriseConfig } = useContext(AppContext);
+  if (!props.location.state?.data) {
+    return <Redirect to={`/${enterpriseConfig.slug}`} />;
+  }
+  return (
+    <Container size="md" className="py-5">
```

Comment:
[curious/clarification] Is this supposed to have the same `Container` `size` as the registration form page (i.e., `size="lg"`?